telemetry/vscode/tst/parserTest.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { validateInput } from '../src/parser'
```

Comment:
I wonder if `parser` should be renamed to `validator`.

Reply:
Yeah it does both so I'm not sure